emeis/core/factories.py

```diff
@@ -7,7 +7,7 @@
 class UserFactory(DjangoModelFactory):
     first_name = Faker("first_name")
     last_name = Faker("last_name")
-    username = Faker("uuid")
+    username = Faker("uuid4")
```

Comment:
Should we really use `uuid4` as a username? I think there is a `user_name` faker that might be better suitable

Reply:
Done!